apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.
+ */
+
+require('babel-polyfill');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Immutable from 'immutable';
+import update from 'react-addons-update';
+import FilterSet from './filterSet';
+import TutorialSet from './tutorialSet';
+
+window.TutorialExplorerManager = function (options) {
+  this.options = options;
+
+  const TutorialExplorer = React.createClass({
+    propTypes: {
+      filterGroups: React.PropTypes.array.isRequired,
+      tutorials: React.PropTypes.array.isRequired,
+      locale: React.PropTypes.string.isRequired
+    },
+
+    getInitialState: function () {
```

Comment:
methods in this file could be of the form `getInitialState()` instead of `getInitialState: function() {`